config/routes.rb

```diff
@@ -90,6 +90,13 @@
     resources :items
     resources :orders, only: [:update]
     resources :merchants do
+      get '/discounts/new', to: 'merchant_discounts#new'
```

Comment:
all routes needed for admin to perform discount crud on a merchant if they wished